authors/apps/articles/serializers.py

```diff
@@ -243,3 +226,19 @@ def handle_existence(instance, data):
                 if instance.favorite \
                 else \
                 serializers.ValidationError("The article is already not in your favorites")
+
+
+class ReportArticleSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = ReportedArticles
+        fields = ['user', 'article', 'message']
+
+    def create(self, validated_data):
+        return ReportedArticles.objects.create(**validated_data)
+
+    def to_representation(self, instance):
+        response = super().to_representation(instance)
+        response["user"] = instance.user.username
+        response["article"] = response["article"] = instance.article.slug
```

Comment:
@byarustev , the second `response["article"]` here is not necessary.

Reply:
sorted